modules/monitoring-giving-view-access-to-a-user.adoc

```diff
@@ -14,14 +14,16 @@ By default, only cluster administrator users and developers have access to metri
 
 .Procedure
 
-* Run this command to give <user> access to all metrics of your services in <namespace>:
+* Run this command to give `<user>` access to all metrics of your services in `<namespace>`:
```

Comment:
I found this a little awkward. What do you think about something like:
> Run this command to give a user access to all metrics of your services in a defined namespace:

Reply:
Good suggestion! Yeah, it was awkward when I came across it.